src/applications/mhv-secure-messaging/util/helpers.js

```diff
@@ -146,6 +146,17 @@ export const titleCase = str => {
 export const httpRegex = /(\b(https?|ftp|file):\/\/[-A-Z0-9+&@#/%?=~_|!:,.;]*[-A-Z0-9+&@#/%=~_|])/gi; // Accepts 'http'
 export const urlRegex = /[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/gi; // Accepts www and https
 
+export const decodeHtmlEntities = str => {
+  if (typeof str !== 'string') return str;
+
+  return str
+    .replace(/&quot;/g, '"') // Replace &quot; with "
+    .replace(/&#x22;/g, '"') // Replace &#x22; with "
+    .replace(/&lt;/g, '<') // Replace &lt; with <
+    .replace(/&gt;/g, '>') // Replace &gt; with >

```

Comment:
this is a drive-by comment, but is there any risk of creating XSS vulnerabilities here? Given that we're dealing with tag characters...

Reply:
@acrollet do you know a security poc who our team can reach out to verify that this doesn't pose a risk?

Reply:
Hello! 

At a first glance, this might be relevant 

https://dsva.slack.com/archives/C5HP4GN3F/p1726495251876289